core/server/services/webhooks/index.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    get listen() {
+        return require('./listen.js');
+    },
+
+    get trigger() {
+        return require('./trigger.js');
+    }
+};
```

Comment:
Do we enforce new line in the end of line? Would be nice to have it, less trouble when doing stuff using non graphical UIs :)

Reply:
Don't think we enforce in lint, but its a good idea 👍